### PR TITLE
Add localization details

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,3 +94,17 @@ Therefore during the second scan we are ignoring few alerts via ZAP rules and th
 During the last scan we are ignoring all the alerts, thus resulting in finding zero alerts. Based on the scan results 
 the actions bot will close the ongoing open issue.
 [![issue closed](./images/zap-issue-3.png)](https://github.com/zaproxy/zaproxy-website/issues/93#issuecomment-611496321)
+
+## Localised Vulnerability details
```
Comment (thc202) [inline README.md:98]: - `Vulnerability` → `Alert`?
 - `Details`

File: README.md [hunk 2]
```diff
@@ -94,3 +94,17 @@ Therefore during the second scan we are ignoring few alerts via ZAP rules and th
...
+
+ZAP is internationalised and vulnerability information is available in many languages.
+
+You can change the language used by this action by changing the locale via the `cmd_options` e.g.:
+    `-z "-config view.locale=fr_FR"`
```
Comment (thc202) [inline README.md:103]: Are the spaces meant for a different rendering?

File: README.md [hunk 3]
```diff
@@ -94,3 +94,17 @@ Therefore during the second scan we are ignoring few alerts via ZAP rules and th
...
+
+This is only only available with the `owasp/zap2docker-weekly` or `owasp/zap2docker-live` docker images.
```
Comment (thc202) [inline README.md:105]: - Add `currently`?
 - `Docker`

**Comment by psiinon** [comment]
Hopefully fixed those.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**